firestore/integration_test/Podfile

```diff
@@ -4,14 +4,14 @@ use_frameworks! :linkage => :static
 
 target 'integration_test' do
   platform :ios, '10.0'
-  pod 'Firebase/Firestore', '9.4.0'
-  pod 'Firebase/Auth', '9.4.0'
+  pod 'FirebaseFirestore', '9.4.0'
```

Comment:
Is this change correct?!

Reply:
We are pinning to the previous version of Firestore, since there is an issue with the newer pods.